packages/@aws-cdk/aws-events-targets/test/ecs/event-rule-target.test.ts

```diff
@@ -403,3 +403,50 @@ test('uses existing IAM role', () => {
     ],
   });
 });
+
+test('uses specified platformVersion', () => {
```

Comment:
I think the test name should mention that it's a fargate task definition with a specific platform version

Reply:
fixes: d608503